app/main.py

```diff
@@ -1,3 +1,54 @@
 class Distance:
     # Write your code here
-    pass
+    def __init__(self, km):
+        self.km = km
+
+    def __str__(self):
+        print(f"Distance: {self.km} kilometers.")
+
+    def __repr__(self):
+        return f"Distance(km={self.km})"
+
+    def __add__(self, second):
+        if isinstance(second, Distance):
+            return Distance(self.km + second.km)
+        elif isinstance(second, int):
+            return Distance(self.km + second)
+
+    def __iadd__(self, second):
+        if isinstance(second, Distance):
+            self.km += second.km
+            return self
+        elif isinstance(second, int):
+            self.km += second
+            return self
+
+    def __mul__(self, other):
+        if isinstance(other, Distance):
```

Comment:
You shouldn't check if **other** is in class **Distance** in **__mul__** magic method